test/manifest.js

```diff
@@ -126,8 +126,8 @@ test('respects directories', async t => {
 	}));
 
 	const MANIFEST = {};
-	MANIFEST[path.join('foo', 'unicorn.css')] = path.join('foo', 'unicorn-d41d8cd98f.css');
-	MANIFEST[path.join('bar', 'pony.css')] = path.join('bar', 'pony-d41d8cd98f.css');
+	MANIFEST[path.join('foo', 'unicorn.css').replace(/\\/g, '/')] = path.join('foo', 'unicorn-d41d8cd98f.css').replace(/\\/g, '/');
+	MANIFEST[path.join('bar', 'pony.css').replace(/\\/g, '/')] = path.join('bar', 'pony-d41d8cd98f.css').replace(/\\/g, '/');
```

Comment:
I don't really know why we're using `path.join` here. We can just remove the joining and specify the paths directly. E.g. `'foo/unicorn.css'`.